charts/opensrp-web/values.yaml

```diff
@@ -72,74 +72,93 @@ recreatePodsWhenConfigMapChange: true
+  EXPRESS_FRONTEND_OPENSRP_CALLBACK_URL: "https://{{ .Values.sharedVars.appDomainName }}/fe/oauth/callback/opensrp"
+  EXPRESS_KEYCLOAK_LOGOUT_URL: "https://{{ .Values.sharedVars.keycloakUrl }}/auth/realms/{{ .Values.sharedVars.keycloakRealm }}/protocol/openid-connect/logout"
+  EXPRESS_LOGS_FILE_PATH: "./logs/error.log"
+  EXPRESS_MAXIMUM_LOGS_FILE_SIZE: "5242880"
+  EXPRESS_MAXIMUM_LOG_FILES_NUMBER: "5"
+  EXPRESS_MAXIMUM_SESSION_LIFE_TIME: "10800"
+  EXPRESS_OAUTH_GET_STATE_URL: "https://{{ .Values.sharedVars.appDomainName }}/oauth/state"
   EXPRESS_OPENSRP_ACCESS_TOKEN_URL: "https://{{ .Values.sharedVars.keycloakUrl }}/auth/realms/{{ .Values.sharedVars.keycloakRealm }}/protocol/openid-connect/token"
+  EXPRESS_OPENSRP_AUTHORIZATION_URL: "https://{{ .Values.sharedVars.keycloakUrl }}/auth/realms/{{ .Values.sharedVars.keycloakRealm }}/protocol/openid-connect/auth"
   EXPRESS_OPENSRP_CALLBACK_URL: "https://{{ .Values.sharedVars.appDomainName }}/oauth/callback/OpenSRP/"
-  EXPRESS_OPENSRP_USER_URL: "https://{{ .Values.sharedVars.opensrpServer  }}/opensrp/user-details/"
   EXPRESS_OPENSRP_CLIENT_ID: "{{ .Values.sharedVars.opensrpClientId }}"
   EXPRESS_OPENSRP_CLIENT_SECRET: "{{ .Values.sharedVars.opensrpClientSecret }}"
-  EXPRESS_SESSION_NAME: "{{ .Values.express.sessionName }}"
-  EXPRESS_OAUTH_GET_STATE_URL: "https://{{ .Values.sharedVars.appDomainName }}/oauth/state"
-  EXPRESS_SESSION_SECRET: "{{ .Values.express.sessionSecret }}"
-  EXPRESS_FRONTEND_OPENSRP_CALLBACK_URL: "https://{{ .Values.sharedVars.appDomainName }}/fe/oauth/callback/opensrp"
-  EXPRESS_REACT_BUILD_PATH: "{{ .Values.express.reactBuildPath }}"
-  NODE_ENV: "{{ .Values.express.nodeEnv }}"
-  EXPRESS_FRONTEND_LOGIN_URL: "{{ .Values.express.frontEndLoginUrl }}"
+  EXPRESS_OPENSRP_LOGOUT_URL: "https://{{ .Values.sharedVars.opensrpServer  }}/opensrp/logout.do"
   EXPRESS_OPENSRP_OAUTH_STATE: "{{ .Values.express.opensrpOauthState }}"
-  EXPRESS_SESSION_PATH: "{{ .Values.express.sessionPath }}"
-  EXPRESS_SESSION_LOGIN_URL: "{{ .Values.express.sessionLoginUrl }}"
-  EXPRESS_FRONTEND_LOGOUT_URL: "{{ .Values.express.sessionFileStorePath }}"
+  EXPRESS_OPENSRP_USER_URL: "https://{{ .Values.sharedVars.opensrpServer  }}/opensrp/user-details/"
+  EXPRESS_PORT: "3000"
   EXPRESS_PRELOADED_STATE_FILE: "{{ .Values.express.preloadedStateFile }}"
-  EXPRESS_MAXIMUM_LOG_FILES_NUMBER: "5"
-  EXPRESS_ALLOW_TOKEN_RENEWAL: "true"
-  EXPRESS_MAXIMUM_SESSION_LIFE_TIME: "10800"
-  EXPRESS_LOGS_FILE_PATH: "{{ .Values.express.logsFilePath }}"
-  EXPRESS_SESSION_FILESTORE_PATH: "{{ .Values.express.sessionFileStorePath }}"
+  EXPRESS_REACT_BUILD_PATH: "{{ .Values.express.reactBuildPath }}"
+  EXPRESS_SERVER_LOGOUT_URL: "https://{{ .Values.sharedVars.appDomainName }}/logout"
+  EXPRESS_SESSION_FILESTORE_PATH: "/tmp/express-sessions"
+  EXPRESS_SESSION_LOGIN_URL: "{{ .Values.express.sessionLoginUrl }}"
+  EXPRESS_SESSION_NAME: "{{ .Release.Name }}-{{ .Release.NameSpace }}"
```

Comment:
Should be `.Release.Namespace `. Does this need to be unique? If not it can be just the release name.

Reply:
@bennsimon dc19960faad15b2ee64fdef8f3cdd309e4f5573a